UFDR parsing failed with email name

I have a Google Warrant Return in UFDR format. When I try to process it IPED fail with the error below. I didn't find any configuration to mitigate this error.

java.lang.Exception: Worker-48 Error while processing Google.ufdr/_DecodedData/Email/Email_dd4b1afa-6f6d-45d6-9fb2-22447751464f (nullbytes)
	at iped.engine.core.Worker.process(Worker.java:188)
	at iped.engine.core.Worker.run(Worker.java:267)
Caused by: java.lang.IllegalArgumentException: DocValuesField "name" is too large, must be <= 32766
	at org.apache.lucene.index.SortedDocValuesWriter.addValue(SortedDocValuesWriter.java:77)
	at org.apache.lucene.index.IndexingChain.indexDocValue(IndexingChain.java:977)
	at org.apache.lucene.index.IndexingChain.processField(IndexingChain.java:758)
	at org.apache.lucene.index.IndexingChain.processDocument(IndexingChain.java:620)
	at org.apache.lucene.index.DocumentsWriterPerThread.updateDocuments(DocumentsWriterPerThread.java:241)
	at org.apache.lucene.index.DocumentsWriter.updateDocuments(DocumentsWriter.java:432)
	at org.apache.lucene.index.IndexWriter.updateDocuments(IndexWriter.java:1532)
	at org.apache.lucene.index.IndexWriter.addDocuments(IndexWriter.java:1503)
	at iped.engine.task.index.IndexTask.process(IndexTask.java:149)...

--- a/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ufed/handler/EmailHandler.java
+++ b/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ufed/handler/EmailHandler.java
@@ -103,10 +103,15 @@ public class EmailHandler extends BaseModelHandler<Email> {
 
     @Override
     public String getTitle() {
+        String title = StringUtils.firstNonBlank(model.getSubject(), model.getId());
+        int maxTitleLen = 4096;
+        if (title != null && title.length() > maxTitleLen) {
+            title = title.substring(0, maxTitleLen);
+        }
         return new StringBuilder()
                 .append("Email") //
                 .append("-[") //
-                .append(StringUtils.firstNonBlank(model.getSubject(), model.getId())) //
+                .append(title) //
                 .append("]") //
                 .toString();
     }

